ios/MaasNearbyTransit/Tests/LinuxMain.swift

```diff
@@ -0,0 +1,7 @@
+import XCTest
+
+import MaasNearbyTransitTests
+
+var tests = [XCTestCaseEntry]()
+tests += MaasNearbyTransitTests.allTests()
+XCTMain(tests)
```

Comment:
Not clear why this file is named `LinuxMain.swift`, is this a convention?

Reply:
This is a created automatically when we're creating new Package 📦, but at the moment i'm deleting these. 🔥